src/tirith/core/core.py

```diff
@@ -218,12 +229,50 @@ def start_policy_evaluation(policy_path: str, input_path: str) -> Dict:
             input_data = json.load(f)
     # TODO: validate input_data using the optionally available validate function in provider
 
-    return start_policy_evaluation_from_dict(policy_data, input_data)
+    # TODO: Move this logic into another module
+    # Merge policy variables into one dictionary
+    var_dicts = []
+    for var_path in var_paths:
+        with open(var_path, encoding="utf-8") as f:
+            var_dicts.append(json.load(f))
+
+    merged_var_dict = _merge_var_dicts(var_dicts)
+
+    variable_pattern = re.compile(r"(?P<var_name>\w+)=(?P<var_json>.+)")
```

Comment:
Research, there might be a standard library function that can process this syntax instead of defining by our own

Reply:
I haven't found anything useful in the standard library so far, marking this as resolved